### PR TITLE
Restructure and phrasing improvements

### PR DESCRIPTION
**Pull request by eadgbear**

### CODE REVIEW

**Review by manjunathprophecy — APPROVED** [review]

File: networking/questions.py [hunk 1]
```diff
@@ -27,10 +32,9 @@ def IngressQuestions(ask=True):
     #     ]
     #     r = questionary.prompt(questions)
     #     result.update(r)
-    return result
 
 
-def EgressQuestions(ask=True):
+"""def EgressQuestions(ask=True):
```
Comment (georgekuruvillak) [inline networking/questions.py:37]: @slyons we are not asking this anymore?

File: ingress_controller/questions.py [hunk 2]
```diff
@@ -1,86 +1,65 @@
...
+            'name': consts.IngressControllerType,
+            'message': consts.IngressControllerTypePromt,
+            'choices': consts.IngressControllerTypeOptions,
+            'when': lambda st: st[consts.IsIngressControllerPresent] == True
+        },
+        {
+            'type': 'text',
+            'name': consts.IngressControllerClass,
+            'message': consts.IngressControllerClassPrompt,
+            'when': lambda st: st[consts.IngressControllerType] in [consts.Nginx, consts.Multiple] == True
```
Comment (georgekuruvillak) [inline ingress_controller/questions.py:29]: this is sweet!! 👍

**Review by georgekuruvillak — COMMENTED** [review]
Regarding the egress question, I have added a comment. Any reason you left it commented?

File: networking/questions.py (hunk 1 above)
Comment (eadgbear) [reply networking/questions.py:37]: It was already commented out: https://github.com/SimpleDataLabsInc/pre-install-script/blob/3b5764e8b73da767901ea3538152818cd5008edb/networking/questions.py#L56

Looks like we're not using it in the helm command either

**Pull request merged**